src/validations/rules/sar.sch

```diff
@@ -223,6 +223,7 @@
             <sch:assert
                 diagnostics="has-security-assessment-plan-resource-diagnostic"
                 doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Results (SAR) §3.5"
+                fedramp:specific="true"
```

Comment:
This is specific to OSCAL (not FedRAMP). All of the inter-document elements are fundamental OSCAL constructs.